pkg/logging/logging.go

```diff
@@ -0,0 +1,103 @@
+// package logging is a small wrapper around github.com/k8snetworkplumbingwg/cni-log
+
+package logging
+
+import (
+	cnilog "github.com/k8snetworkplumbingwg/cni-log"
+)
+
+const (
+	labelCNIName     = "cniName"
+	labelContainerID = "containerID"
+	labelNetNS       = "netns"
+	labelIFName      = "ifname"
+	cniName          = "sriov-cni"
+)
+
+var (
+	logLevelDefault = cnilog.ErrorLevel
+	containerID     = ""
+	netNS           = ""
+	ifName          = ""
+)
+
+func init() {
+	SetLogFile("")
+	SetLogLevel("")
+}
+
+// SetLogLevel sets the log level to either verbose, debug, info, warn, error or panic. If an invalid string is
+// provided, it uses error.
+func SetLogLevel(l string) {
+	ll := cnilog.StringToLevel(l)
+	if ll == cnilog.InvalidLevel {
+		ll = logLevelDefault
+	}
+	cnilog.SetLogLevel(ll)
+}
+
+// SetLogFile sets the log file for logging. If the empty string is provided, it uses stderr.
+func SetLogFile(fileName string) {
+	if fileName == "" {
+		cnilog.SetLogStderr(true)
+		cnilog.SetLogFile("")
+		return
+	}
+	cnilog.SetLogFile(fileName)
+	cnilog.SetLogStderr(false)
```

Comment:
so, is it exclusive? only file or only stderr, but not both?

Reply:
yeah that was my idea to have an "either log to file or log to stderr". The other model would be: "log to stderr and optionally log to file as well" .. or yet another option would be to expose the stderr switch to users as well and have 3 options. But I find the either/or one most intuitive. That said, I'm open to changing it if you strongly favor something else

Reply:
No, it's fine like it is. Just wanted to check if it was really intended.